docs/commands.en.md

```diff
@@ -8,6 +8,7 @@ You can use these commands globally at any time.
 | ------ | -------- |
 | `/achievements`, `/ach`         | Lists an overview of all achievements |
 | `/bannotify`                    | Deactivate notifications concerning your ban days |
+| `/discord unverify`             | Separate the verification and connection between your Minecraft and Discord account. |
```

Comment:
ist Separate das richtige wort in dem kontext? bin kein englisch experte 😅

Reply:
Vllt wäre "Remove" oder "Revoke" ein besseres Wort?

Reply:
Habe halt auf Deutsch Trennt geschrieben, kann es aber auch auf Remove anpassen